src/cljs/org/broadinstitute/firecloud_ui/page/workspaces_list.cljs

```diff
@@ -194,7 +196,7 @@
          :else
          [:div {:style {:margin "0 2em"}}
           [WorkspaceTable
-           (assoc props :workspaces workspaces :billing-projects billing-projects)]])))
+           (assoc props :workspaces workspaces :billing-projects billing-projects :disabled-reason error-message)]])))
```

Comment:
I don't think this is right.  The error message is kind of an infodump; this flag was designed to be a keyword (`:no-billing` etc.)